process/tags/svchost_windows.go

```diff
@@ -82,6 +82,8 @@ func (h *SVCHostTagHandler) CreateProfile(p *process.Process) *profile.Profile {
 			return profile.New(&profile.Profile{
 				Source:              profile.SourceLocal,
 				Name:                "Windows Service: " + osdetail.GenerateBinaryNameFromPath(tag.Value),
+				Icon:                `C:\Windows\System32\@WLOGO_48x48.png`,
+				IconType:            profile.IconTypeFile,
 				UsePresentationPath: false,
```

Comment:
That one is not yet supported by the UI since that would require electron to actually load that file. `blob` and `database` are supported. Maybe we should add `file` support using a Portmaster API?